src/index.jsx

```diff
@@ -58,6 +58,7 @@ initialize({
         ALGOLIA_JOBS_INDEX_NAME: process.env.ALGOLIA_JOBS_INDEX_NAME || null,
         ALGOLIA_PRODUCT_INDEX_NAME: process.env.ALGOLIA_PRODUCT_INDEX_NAME || null,
         ALGOLIA_SEARCH_API_KEY: process.env.ALGOLIA_SEARCH_API_KEY || null,
+        SEARCH_CATALOG_URL: process.env.SEARCH_CATALOG_URL || null,
```

Comment:
nit: The name `Catalog` has a loaded meaning, as the alternative name for the Course-Discovery IDA is "Catalog". I think it's a bit clearer if the name of this setting refers to the marketing site in some way? 

This is not a hill I'm going to die on and feel free to ignore me if you feel otherwise.

Reply:
This makes sense. We can't modify the name as it appears in the `.env` files as that is determined by the common config. But we can modify it here where you have highlighted so that whenever it's referenced in the app, the name is `MARKETING_SITE_SEARCH_URL`. I've made those changes 👍